src/platform/3ds/main.c

```diff
@@ -476,21 +472,33 @@ static u32 _setupTex(int out, bool faded) {
 static void _drawTex(struct mCore* core, bool faded, bool both) {
 	unsigned screen_w, screen_h;
 	bool isWide = screenMode >= SM_PA_TOP && gfxIsWide();
+
+	if (filterMode < FM_LINEAR_1x || filterMode > FM_LINEAR_3x) {
+		// Other filtering modes are not supported
+		filterMode = FM_LINEAR_3x;
+	}
```

Comment:
I wouldn't call these "other", I'd call them "out of range", since the enum is specific to 3DS already.